hyperspy/_signals/eels.py

```diff
@@ -299,6 +299,31 @@ def estimate_zero_loss_peak_centre(s, mask, signal_range):
             substract_from_offset(without_nans(zlpc.data).mean(),
                                   also_align + [self])
 
+    def spikes_removal_tool(self, signal_mask=None,
+                            navigation_mask=None,
+                            signal_start=None,
+                            signal_end=None,
+                            filter_zero_loss=False,
+                            zero_loss_width=5.0):
+        if filter_zero_loss:
```

Comment:
This option is really useful! Could implemented without the ``signal_start`` and ``signal_end`` kwargs?